### PR TITLE
pd: parse ALREADY_BOOTSTRAPPED error.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: tests/pd/mock/mocker/bootstrap.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+use kvproto::pdpb::*;
+
+use super::Mocker;
+use super::Result;
+use super::DEFAULT_CLUSTER_ID;
+
+#[derive(Debug)]
+pub struct Bootstrap {}
+
+impl Bootstrap {
```
Comment (siddontang) [inline tests/pd/mock/mocker/bootstrap.rs]: AlreadyBootstrap

**Comment by siddontang** [comment]
can we test bootstrap_cluster in node.rs?

**Comment by disksing** [comment]
@siddontang  In node.rs, tests are using `TestPdClient` which returns `Error::ClusterBootstrapped` correctly, see https://github.com/pingcap/tikv/blob/083b54a39b69a81158f1946c5ebc89fb3d5a7807/tests/raftstore/pd.rs#L502 , so I  don't think we need to add another test for it.

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay @overvenus

**Comment by zhangjinpeng1987** [comment]
LGTM

File: tests/pd/test_rpc_client.rs [hunk 2]
```diff
@@ -69,6 +69,30 @@ fn test_rpc_client() {
 }
 
 #[test]
+fn test_reboot() {
+    let mut eps = vec![
+    "http://127.0.0.1:52730".to_owned(),
```
Comment (BusyJay) [inline tests/pd/test_rpc_client.rs]: Should indent 4 more space.

File: tests/pd/test_rpc_client.rs [hunk 3]
```diff
@@ -69,6 +69,30 @@ fn test_rpc_client() {
...
+
+    let se = Arc::new(Service::new(eps.clone()));
+    let lc = Arc::new(AlreadyBootstrap::new());
+
+    let _server = MockServer::run("127.0.0.1:52730", se.clone(), Some(lc.clone()));
+    thread::sleep(Duration::from_secs(1));
+
+    let client = RpcClient::new(&eps.pop().unwrap()).unwrap();
+
+    assert_eq!(client.is_cluster_bootstrapped().unwrap(), false);
```
Comment (BusyJay) [inline tests/pd/test_rpc_client.rs]: Use `assert` instead.

**Pull request merged**